settings/src/lib.rs

```diff
@@ -86,6 +86,14 @@ fn default_tunnel_timeout() -> u64 {
     900 // 15 minutes
 }
 
+fn default_local_fee() -> u32 {
+    0
+}
+
+fn default_metric_factor() -> u32 {
+    1000u32
```

Comment:
um souldn't the factor be a float now that we're doing logs and float operations in babel?

Reply:
Currently the value corresponds to thousandths of the factor. I'm not using a `f64` so that the `Eq` derive still works (not defined for `f64` so that  people only compare them consciously I presume). I agree that what I did feels kinda complicated and I'm more than open to alternatives